### PR TITLE
Add more features and another breaking change

### PR DESCRIPTION
**Pull request by 304NotModified**

### CODE REVIEW

**Comment by snakefoot** [comment]
Not sure the change of `SimpleLayout.ToString()` is relevant as breaking change. Notice I also changed the ToString of NLog LoggingConfiguration :) And I also changed the output of the NLog InternalLogger several places.

**Review by snakefoot — APPROVED** [review]

File: _posts/2021-08-25-nlog-5-0-preview1-ready.md [hunk 1]
```diff
@@ -187,6 +187,31 @@ Having the logging-layer making calls into the dependency-injection-layer can ca
 or deadlocks. Both issues are extremely annoying to debug and diagnose, so one should be very careful to ensure interfaces
 are implemented with singleton-lifetime.
 
+### Aliases support for targets, layouts, layout renderers and conditions
+
+The following attributes are now allowed multiple times for a class:
```
Comment (snakefoot) [inline _posts/2021-08-25-nlog-5-0-preview1-ready.md:192]: Maybe explain that is a feature for users that are creating their own custom NLog extensions, that they now have the option to define additional alias upfront.

File: _posts/2021-08-25-nlog-5-0-preview1-ready.md [hunk 2]
```diff
@@ -187,6 +187,31 @@ Having the logging-layer making calls into the dependency-injection-layer can ca
...
+The following aliases are added: 
+- The Mail Target has now the aliases Email, SMTP and SMTP-Client
+- The Trace Target has now the alias TraceSystem
+- ${Event-properties} has now the alias {Event-property}
+- ${logger} has now the alias ${loggername}
+
+All aliases are also listed and searchable on https://nlog-project.org/config/
+
+### Dashes (-) will be ignored when parsing names
+Dashes in names in targets, layout renderers, layouts, filters are ignored. For example: `${loggername}` could be written als `${logger-name}`, and ColoredConsole could be written als Colored-Console.
```
Comment (snakefoot) [inline _posts/2021-08-25-nlog-5-0-preview1-ready.md:210]: Maybe explain that it is the symbol-name / type-name that now has more relaxed handling, when using in NLog Configuration files.

**Comment by 304NotModified** [comment]
@snakefoot 

FYI, I made these packages obsolete in NuGet:

- https://www.nuget.org/packages/NLog.Config/
- https://www.nuget.org/packages/NLog.Extended/

File: _posts/2021-08-25-nlog-5-0-preview1-ready.md [hunk 3]
```diff
@@ -206,6 +206,14 @@ The following aliases are added:
 
 All aliases are also listed and searchable on https://nlog-project.org/config/
 
+Creating your own extension? So this is now allowed:
```
Comment (snakefoot) [inline _posts/2021-08-25-nlog-5-0-preview1-ready.md:209]: Would probably move this sentence to the top.

**Pull request merged**